AR-js-org/AR.js
# Add latLonToWorld() method to gps-new-camera for consistency with gps…

…-projected-camera



**⚠️ All PRs have to be done versus 'dev' branch, so be aware of that, or we'll close your issue ⚠️**

**What kind of change does this PR introduce?**


Adds the `latLonToWorld()` method to `gps-new-camera` for consistency with `gps-projected-camera`.

**Can it be referenced to an Issue? If so what is the issue # ?**

**How can we test it?**



Create a project making use of `gps-new-camera` and access the entity from JavaScript. Call `latLonToWorld()`, passing in the latitude 51.05 and longitude -0.72. The array returned should contain values at around -80000 and -6600000.

**Summary**


**Does this PR introduce a breaking change?**

No

**Please TEST your PR before proposing it. Specify here what device you have used for tests, version of OS and version of Browser**

Chrome on desktop Linux and Android 12.

**Other information**
